src/differentiation/jaches_products.jl

```diff
@@ -198,112 +198,105 @@ end
 
 ### Operator Forms
 
-struct JacVec{F, T1, T2, xType}
+mutable struct FwdModeAutoDiffVecProd{F,U,C,V,V!} <: AbstractAutoDiffVecProd
```

Comment:
use the original names

Reply:
The diff isn't conveying things correctly - you might want to look at the files. `FwdModeAutoDiffVecProd, RevModeAutoDiffVecProd` are not user facing structs. They are not exported.

What are exported are the functions `JacVec, HesVec, HesVecGrad, VecJac`. These are the original names.

eg JacVec is
https://github.com/JuliaDiff/SparseDiffTools.jl/blob/4d7ce81bf3663e742dadfd560b6697b9edb7052f/src/differentiation/jaches_products.jl#L226-L250

Reply:
@ChrisRackauckas